test/correctconfig.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
```

Comment:
nitpick: Naming of this file could potentially reflect what it contains, it's currently just "correct config" but you could consider `passing_test.deployment.yaml` to be more expressive and explicit.